src/templates/ObservabilityPackDetails.js

```diff
@@ -37,6 +37,169 @@ const allowedElements = [
   'hr',
 ];
 
+const renderDashboards = (dashboards) => {
+  const content = dashboards.map((dashboard) => (
+    <>
+      <h3>{dashboard.name}</h3>
+      <ImageSlider height={400} images={dashboard.screenshots} />
+      {dashboard.description && (
+        <>
+          <h4>Description</h4>
+          <p>{dashboard.description}</p>
+        </>
+      )}
+    </>
+  ));
+
+  return (
+    <>
+      <Intro
+        css={css`
+          margin-bottom: 16px;
+        `}
+      >
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+        commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+        velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+        occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+        mollit anim id est laborum.
+      </Intro>
+      {content}
+    </>
+  );
+};
+
+const renderAlerts = (alerts) => {
+  const alertContent = alerts.map((alert) => (
+    <>
+      <h3>{alert.name}</h3>
+      {alert.description && (
+        <>
+          <h4>Description</h4>
+          <p>{alert.description}</p>
+        </>
+      )}
+    </>
+  ));
+
+  return (
+    <>
+      <Intro
+        css={css`
+          margin-bottom: 16px;
+        `}
+      >
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+        commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+        velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+        occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+        mollit anim id est laborum.
+      </Intro>
+      {alertContent}
+    </>
+  );
+};
+
+const renderSynthetics = () => {
+  return (
+    <Intro
+      css={css`
+        margin-bottom: 16px;
+      `}
+    >
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+      commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+      velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+      cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
+      est laborum.
+    </Intro>
+  );
+};
```

Comment:
These all look fairly similar. Perhaps we could abstract this out into it's own component. Something like `ObservabilityPackEmptyState`?

Reply:
Oh, I misunderstood, this is the _non-empty_ state? Either way, we might have little too much duplication in this file.

Reply:
I agree, I just didn't want to abstract it since the copy will be different for each state once that work gets done. Do you think it would be worth abstracting for now?

Reply:
Specifically the changes in this PR https://github.com/newrelic/developer-website/pull/1478

Reply:
@zstix Can you take a look at this now that the unique copy has gone in for each tab? I didn't have any ideas of the top of my head for abstracting a tab page since each one requires different text and portions of the pack itself.